client/components/AdoptionForm.js

```diff
@@ -0,0 +1,98 @@
+import React,{useState} from "react"
+
+const AdoptionForm =()=>{
+const[adoptionData, setAdoptionData] = useState({
```

Comment:
Please indent everything inside your component to reflect that it is inside your component!